home-data/src/main/java/com/softserveinc/ita/homeproject/homedata/repository/apartment/ApartmentRepository.java

```diff
@@ -1,6 +1,6 @@
-package com.softserveinc.ita.homeproject.homedata.repository;
```

Comment:
I think that those packages in which there is only one repository should not be created. It seems to me better to create packages with a more general purpose, e.g.:
- package "cooperation_element" - includes ApartmentRepository, ContactRepository, CooperationRepository, HouseRepository, OwnershipRepository, UserRepository;
- "authorities" package - PermissionRepository, RoleRepository, UserCooperationRepository.
And to leave NewsRepository in the "repository" package root.